src/params/immunity.jl

```diff
@@ -22,6 +22,7 @@ end
 
 rawinfectivity(table::StrainImmunityTable, strain::StrainKind) = table[UInt(1), UInt(strain)]
 condinfectivity(table::StrainImmunityTable, immunity::ImmunityState, strain::StrainKind) = table[UInt(immunity), UInt(strain)] / table[UInt(1), UInt(strain)]
+vaccinated(immunity::ImmunityState) = immunity == VecVacImmunity || immunity == MRNAVacImmunity
```

Comment:
I don't think it is needed also the natural immunity should have similar effect

Reply:
I can replace functions `vaccinated()` with `immunity > NoImmunity` ?

Reply:
maybe `immunity != NoImmunity` I would try to avoid depending on order of enums

Reply:
ok, i will change it tomorrow.